Index: src/library/Library.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>public class Library {\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/library/Library.java b/src/library/Library.java
--- a/src/library/Library.java	(revision 813fb831b2a06919c1df76884e0706e844904641)
+++ b/src/library/Library.java	(date 1611749510843)
@@ -1,2 +1,21 @@
+import java.util.Scanner;
+
 public class Library {
+
+    public void addBook() {
+        Scanner input = new Scanner(System.in);
+        System.out.println("Enter book title: ");
+        String bookTitle = input.nextLine();
+
+        System.out.println("Enter author: ");
+        String author = input.nextLine();
+
+        System.out.println("Enter genre: ");
+        String genre = input.nextLine();
+
+        bookCollection.add(bookTitle, new Book(bookTitle,author,genre,true));
+
+    }
+
+
 }
